drop-storage/src/lib.rs

```diff
@@ -49,7 +50,7 @@ impl Storage {
             "INSERT OR IGNORE INTO peers (id) VALUES (?1)",
             transfer.peer
         )
-        .execute(&mut conn)
+        .execute(conn.acquire().await?)
```

Comment:
It should probably be `&mut *conn`

Reply:
It shouldn't compile if it would need a dereference?

Reply:
Oh, I get it, it's probably that referencing allows me to omit `acquire()` part?

Reply:
So I've tried it and yep `&mut *conn` works as well and looks a bit nicer so changed all occurences